app/digests/api.py

```diff
@@ -1,26 +1,54 @@
+import json
+import os
 from typing import Any, Dict
 
 from django.conf import settings
-from rest_framework import viewsets
+from elife_api_validator import SCHEMA_DIRECTORY
+from jsonschema import validate as validate_json
+from jsonschema import ValidationError
+from rest_framework import viewsets, status
+from rest_framework.request import Request
 from rest_framework.response import Response
 
+from digests.exceptions import generate_error_string
 from digests.models import Digest
 from digests.pagination import DigestPagination
 from digests.serializers import DigestSerializer
 
 
+def get_schema(schema_name: str) -> Dict:
+    with open(os.path.join(SCHEMA_DIRECTORY, schema_name)) as schema:
+        val = json.loads(schema.read())
+        return val
+
+
 class DigestViewSet(viewsets.ModelViewSet):
     model = Digest
     queryset = Digest.objects.all()
     serializer_class = DigestSerializer
     pagination_class = DigestPagination
 
     content_type = settings.DIGESTS_CONTENT_TYPE
+    schema_name = 'digest.v1.json'
 
     def _create_response(self, data: Dict[str, Any]) -> Response:
         return Response(data, content_type=self.content_type)
 
-    def list(self, request, *args, **kwargs):
+    def create(self, request: Request, *args, **kwargs) -> Response:
+        try:
+            validate_json(request.data, schema=get_schema(self.schema_name))
+            serializer = self.get_serializer(data=request.data)
+            serializer.is_valid(raise_exception=True)
+            serializer.save()
+            headers = self.get_success_headers(serializer.data)
+            return Response(serializer.data, status=status.HTTP_201_CREATED, headers=headers)
+        except ValidationError as err:
+            return Response(
+                {'error': generate_error_string(message=err.message, path=err.path)},
```

Comment:
Errors need to comply with RFC 7807. Take a look at https://github.com/elifesciences/profiles/blob/b1ea49ca94022bdbf8d36d21773054b278ed3d09/profiles/api/errors.py#L55-L65.